src/Symfony/Component/Workflow/README.md

```diff
@@ -4,7 +4,7 @@ Workflow Component
 Resources
 ---------
 
-  * [Documentation](https://symfony.com/doc/current/components/workflow/introduction.html)
+  * [Documentation](https://symfony.com/doc/master/components/workflow.html)
```

Comment:
That would lead to another 404 page until Symfony 3.2 is released. Is that what you want?

I suggest to keep "master" until 3.2 is released and then update this link again.